src/psyclone/psyGen.py

```diff
@@ -5986,6 +5994,9 @@ def _part_ref_handler(self, node, parent):
                 argument = node.items[1].items[0]
                 if len(node.items[1].items) > 1:
                     # If it has more than a single argument create a CodeBlock
+                    # TODO: Note that real(var, kind) expressions are not
+                    # supported because Fortran kinds are still not captured
+                    # (Issue #375)
```

Comment:
Please could you add a suitable message to the `NotImplementedError`

Reply:
The Error will be caught by the caller and trigger the creation of a CodeBlock (it won't produce and error message). I added a description in the doctring for this error (which was missing :))

Reply:
Ah yes, I'll claim that's why I misunderstood ;-)